commands/_setAngryPoints.js

```diff
@@ -0,0 +1,34 @@
+/*CMD
+  command: /setAngryPoints
+  help: 
+  need_reply: 
+  auto_retry_time: 
+  folder: 
+  answer: 
+  keyboard: 
+  aliases: 
+CMD*/
+
+if(!user){return}
+//allowed to run through admin panel only
+
+var angryPointsPanel = AdminPanel.getPanel("AngryPoints")
```

Comment:
you can use here: https://help.bots.business/scenarios-and-bjs/admin-panel#getting-all-fields-values-from-panel

var values = AdminPanel.getPanelValues("AngryPoints")

then you can use
values.masterId and etc

it is more simple